bin/test-scratch

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -eu
+
+while :; do
+  while getopts ":a:e:p:s:" arg; do
+    case $arg in
+      a)
+        auth_file_key="$OPTARG"
+        ;;
+      e) 
+        enc_auth_file="$OPTARG"
+        ;;
+      p)
+        source_path="$OPTARG"
+        ;;
+      s)
+        scratch_def_file="$OPTARG"
+        ;;
+      *)
+        echo Ignoring unknown option: $OPTARG
+        ;;
+    esac
+  done
+  OPTIND=$((OPTIND+1))
+  [ $OPTIND -gt $# ] && break
+done
+
+# Decrypt Auth URL File
+openssl enc -d -aes-256-cbc -md md5 -in ${enc_auth_file} -out auth_url.txt -k ${auth_file_key}
```

Comment:
The `-md md5` flag here is not great and is a result of defaults from the macOS openssl different from that of ubuntu. Longterm we should aim to migrate all encrypted files from md5 to sha512 and then update this.

Reply:
Lets open a ticket for this